controllers/cloud.redhat.com/providers/sidecar/provider.go

```diff
@@ -4,7 +4,8 @@ import (
 	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers"
 )
 
-var DefaultImageSideCarTokenRefresher = "quay.io/observatorium/token-refresher:master-2023-09-20-f5e3403" // nolint:gosec
+var DefaultImageSideCarTokenRefresher = "quay.io/observatorium/token-refresher:master-2022-10-21-a99ce82" // nolint:gosec
```

Comment:
Take a look at the other implementations which use default images - we should have a way to override these in config, and even ClownEnv in some places

Reply:
added